### ISSUE TITLE
Loss and labels

### ISSUE THREAD

**Issue by DebanjanaKar** [pengbaolin/SC-GPT#15]
In the paper, in section 3, there is a mention of the loss function only attending to the utterances and not the intent during training. "During training, the prediction loss is only computed for x0, and A0 provides the attended conditions."  But I look at the labels provided for training, it includes the intent as well. Can you please help me understand how the code achieves to compute loss only for the utterance leaving out the intent? Any help would really be appreciated.
